ui/src/config/section/compute.js

```diff
@@ -125,6 +125,7 @@ export default {
           show: (record) => { return ['Running'].includes(record.state) },
           args: (record, store) => {
             var fields = []
+            fields.push('forced')
```

Comment:
@sureshanaparti what about systemvms, VR?

Reply:
> @sureshanaparti what about systemvms, VR?

The reboot for systemVMs and VR are performed using _rebootSystemVm_ and _rebootRouter_ APIs respectively, should add support for these as well ?

Reply:
Added 'forced' reboot option for System VM and Router, as well. Existing reboot router operation already stops and starts the router, so the new 'forced' flag, would force stop and then start the router.